docs/topics/basics/keyspaces/README.md

```diff
@@ -23,13 +25,32 @@ CassFuture* use_future
 /* Check future result... */
 ```
 
-## Single session mulitple keyspaces
+## Single Session and Mulitple Keyspaces
 
-It is possible to use multiple keyspaces from a single session by full qualifying the table names in your queries e.g. `keyspace_name.table_name`.
+It is possible to interact with multiple keyspaces using a single session object by fully qualifying the table names in your queries e.g. `keyspace_name.table_name`.
 
-Examples:
+### Examples
 
-```cql
+```sql
```

Comment:
"cql" should be supported or it will be. (https://twitter.com/patrickmcfadin/status/557977077903663104)